diff --git a/xformers/ops/fmha/common.py b/xformers/ops/fmha/common.py
index e510e77..dfb6200 100644
--- a/xformers/ops/fmha/common.py
+++ b/xformers/ops/fmha/common.py
@@ -250,7 +250,7 @@ class AttentionOpBase(BaseOperator):
 
     OPERATOR: Any
     SUPPORTED_DEVICES: Set[str]
-    CUDA_MINIMUM_COMPUTE_CAPABILITY: Tuple[int, int] = (5, 0)
+    CUDA_MINIMUM_COMPUTE_CAPABILITY: Tuple[int, int] = (3, 7)  # Tesla K80 (sm_37) base default
     SUPPORTED_DTYPES: Set[torch.dtype]
     SUPPORTED_MAX_K: float
     SUPPORTED_ATTN_BIAS_TYPES: Set[Any] = {type(None)}
