diff --git a/xformers/csrc/attention/cuda/fmha/generate_kernels.py b/xformers/csrc/attention/cuda/fmha/generate_kernels.py
index 661a037..8c2b58f 100644
--- a/xformers/csrc/attention/cuda/fmha/generate_kernels.py
+++ b/xformers/csrc/attention/cuda/fmha/generate_kernels.py
@@ -20,7 +20,7 @@ DTYPES = {
     "bf16": "cutlass::bfloat16_t",
 }
 
-SM = [50, 70, 75, 80, 100]  # Sm80 kernels support up to Sm100
+SM = [37, 50, 70, 75, 80, 100]  # Sm80 kernels support up to Sm100; Sm37 added for Tesla K80
 
 KERNEL_IMPL_TEMPLATE = """__global__ void __launch_bounds__(
     {CPP_CLASS}::kNumThreads,
